hibernate-core/src/main/java/org/hibernate/sql/results/graph/basic/BasicResultAssembler.java

```diff
@@ -26,10 +26,16 @@ public static <X> BasicResultAssembler<X> from(SqlSelection selection, JavaType<
 		return new BasicResultAssembler<>( selection.getValuesArrayPosition(), javaType );
 	}
 
+	private static final ResultsLogger RESULTS_LOGGER = ResultsLogger.RESULTS_MESSAGE_LOGGER;
+
 	private final int valuesArrayPosition;
 	private final JavaType<J> assembledJavaType;
 	private final BasicValueConverter<J,?> valueConverter;
 
+	//We read this only once for each new instance of BasicResultAssembler:
+	//an assembler is short lived.
+	private final boolean RESULTS_LOGGER_DEBUG_ENABLED = RESULTS_LOGGER.isDebugEnabled();
```

Comment:
org.hibernate.sql.results.ResultsLogger#DEBUG_ENABLED

Reply:
ah interesting, you made them constants? Don't want to allow for the level to be changed at runtime?

Reply:
ic - since I'm a bit confused about this I'll remove this change from the current PR so I can send it separately:
the other changes are more urgent, while this one was I thought a trivial one - my bad I shouldn't have incorporated two JIRAs in a single PR.